jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardUtilitiesTests.java

```diff
@@ -0,0 +1,112 @@
+
+    @DataProvider(name = "cardUtilitiesElementsWithWidth")
+    public static Object[][] cardUtilitiesElementsWithWidth() {
+        return new Object[][]{
+                {cardWidth75, 75, "212.4px"},
+                {cardWidth50, 50, "141.6px"}
+        };
+    }
+
+    @DataProvider(name = "cardUtilitiesElementsWithTitle")
+    public static Object[][] cardUtilitiesElementsWithTitle() {
+        return new Object[][]{
+                {cardWidth75, "SPIDER MAN (W-75)"},
+                {cardWidth50, "HULK\n(W-50)"}
+        };
+    }
+
+    @DataProvider(name = "cardUtilitiesElementsWithText")
+    public static Object[][] cardUtilitiesElementsWithText() {
+        return new Object[][]{
+                {cardWidth75, "Spider-Man is a fictional superhero created by writer-editor Stan Lee and writer-artist Steve Ditko."},
+                {cardWidth50, "The Hulk is a fictional superhero appearing in publications by the American publisher Marvel Comics."}
+        };
+    }
+
+    @DataProvider(name = "cardUtilitiesElementsWithLinks")
+    public static Object[][] cardUtilitiesElementsWithLinks() {
+        return new Object[][]{
+                {cardWidth75, "https://en.wikipedia.org/wiki/Spider-Man"},
+                {cardWidth50, "https://en.wikipedia.org/wiki/Hulk_(film)"}
+        };
+    }
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test(dataProvider = "cardUtilitiesElements")
+    public void baseValidationTest(CardUtilities cardUtilitiesElem) {
+        baseValidation(cardUtilitiesElem);
+        cardUtilitiesElem.unhighlight();
+    }
+
+    @Test(dataProvider = "cardUtilitiesElementsWithWidth")
+    public void cardValidationTest(CardUtilities cardUtilitiesElem, int widthInPercent, String widthInPixels) {
+        cardUtilitiesElem.core().is()
+                .attr("class", String.format("card w-%d", widthInPercent))
+                .css("width", widthInPixels)
+                .css("background-color", "rgba(255, 255, 255, 1)");
+    }
+
+    @Test(dataProvider = "cardUtilitiesElementsWithTitle")
+    public void titleValidationTest(CardUtilities cardUtilitiesElem, String title) {
+        cardUtilitiesElem.cardTitle.core().is()
+                .tag("h5")
+                .text(title)
+                .css("margin-bottom", "12px");
+    }
+
+    @Test(dataProvider = "cardUtilitiesElementsWithText")
+    public void textValidationTest(CardUtilities cardUtilitiesElem, String text) {
+        cardUtilitiesElem.cardText.core().is()
+                .tag("p")
+                .text(text);
+    }
+
+    @Test(dataProvider = "cardUtilitiesElementsWithLinks")
+    public void buttonValidationTest(CardUtilities cardUtilitiesElem, String link) {
+        cardUtilitiesElem.cardButton.core().is()
+                .text("Read more")
+                .css("color", "rgba(255, 255, 255, 1)")
+                .css("background-color", "rgba(0, 123, 255, 1)")
```

Comment:
I suggest to put background-color and other such names that used a few times to constant strings. It allows as to change it in one place if we will need to change it instead of changing it in several places

Reply:
fixed